rust/lib/src/lib.rs

```diff
@@ -9,7 +9,8 @@ use android_logger::{Config, FilterBuilder};
 #[cfg(target_os = "android")]
 use log::Level;
 
-use base64::{decode, encode};
+use base64::Engine;
+use base64::engine::general_purpose::STANDARD;
```

Comment:
I need to use STANDARD because swift & kotlin are expecting the data encoded with `padding`, and also we avoid using `decode` & `encode` from base64 which are deprecated.